### DISCUSSION TITLE
Duda en orden coordenadas IN-RECTANGLE y IN-CIRCLE

### DISCUSSION THREAD

**Discussion by bugfly1** [IIC2133-PUC/2024-2#148] [Tarea 2] [answered]
El enunciado menciona que el resultado se debe ordenar por el valor de $x,y$. ¿Esto significa que para un par de pinguinos con coordenadas $(x_1, y_1)$ y $(x_2, y_2)$ se tendrian que comparar $x_1 < x_2$ y luego $y_1 < y_2$? ¿No sería redundante ordenarlo por el valor de $y$ en el caso de empate si este es el caso?

Si este no es el ordenamiento esperado, agradecería una aclaración al respecto :)

**Comment by sebacarrascop** [comment, marked as answer]
La idea es que se ordene por 2 llaves, primero por eje X, y en caso de que existan más elementos en una misma posición de X, esos se ordenan por eje Y. Además nunca habrá un doble empate ya que se menciona en los supuestos que solo habrá como máximo un pingüino en cada punto. Espero que haya quedado claro, saludos.
